bin/ci/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+bundle config set path $PWD/vendor/bundle
+bundle install --jobs=3 --retry=3
+
+if ! which zeus &>/dev/null; then
```

Comment:
You can drop these three lines in this case. For super_diff I'm using zeus in CI to speed up integration tests, but we don't have that for shoulda-matchers.

Reply:
EDIT: Well, we do use Zeus for development, but it's more of an optional thing, and we don't make use of it for CI.

Reply:
Done!